static/app/views/alerts/rules/details/metricHistory.tsx

```diff
@@ -0,0 +1,244 @@
+            </Tooltip>
+          </StatusLevel>
+          <Link
+            to={{
+              pathname: alertDetailsLink(organization, incident),
+              query: {alert: incident.identifier},
+            }}
+          >
+            {tct('#[id]', {id: incident.identifier})}
+          </Link>
+        </Title>
+        <Cell>
+          {tct('[title] [selector] [threshold]', {
+            title:
+              AlertWizardAlertNames[getAlertTypeFromAggregateDataset(incident.alertRule)],
+            selector:
+              incident.alertRule.thresholdType === AlertRuleThresholdType.ABOVE
+                ? 'above'
+                : 'below',
+            threshold:
+              currentTrigger === 'Warning' ? warningThreshold : criticalThreshold,
+          })}
+        </Cell>
+        <Cell>{threshold}</Cell>
+        <Cell>
+          <IconWrapper>
+            <Tooltip title={t('People who have viewed this')} skipWrapper>
+              <StyledIconShow size="xs" color="gray200" />
+            </Tooltip>
+            {incident.seenBy.length}
+          </IconWrapper>
+        </Cell>
+        <StyledDateTime date={incident.dateCreated} />
+      </ErrorBoundary>
+    );
+  };
+
+  return (
+    <CollapsePanel
+      items={incidents!.length}
+      collapseCount={COLLAPSE_COUNT}
+      buttonTitle={tn(
+        'Hidden Alert',
+        'Hidden Alerts',
+        incidents!.length - COLLAPSE_COUNT
+      )}
+    >
+      {({isExpanded, showMoreButton}) => (
+        <React.Fragment>
+          <StyledPanelTable
+            headers={[
+              t('Alert'),
+              t('Reason'),
+              t('Duration'),
+              t('Seen By'),
+              t('Date Triggered'),
+            ]}
+            isEmpty={!incidents!.length}
+            emptyMessage={t('No alerts triggered during this time.')}
+            expanded={incidents!.length < COLLAPSE_COUNT || isExpanded}
+          >
+            {incidents!.map((incident, idx) => {
+              if (idx >= COLLAPSE_COUNT && !isExpanded) {
+                return null;
+              }
+              return renderActivity(incident);
+            })}
+          </StyledPanelTable>
+          {incidents!.length > COLLAPSE_COUNT && (
+            <ShowMoreButton expanded={isExpanded}>{showMoreButton}</ShowMoreButton>
+          )}
+        </React.Fragment>
+      )}
+    </CollapsePanel>
+  );
+}
+
+export default MetricHistory;
+
+const dateTimeCss = p => css`
+  color: ${p.theme.gray300};
+  font-size: ${p.theme.fontSizeMedium};
+  display: flex;
+  justify-content: center;
+`;
+
+const StyledPanelTable = styled(PanelTable)<{expanded: boolean; isEmpty: boolean}>`
+  grid-template-columns: max-content 3fr repeat(3, max-content);
+
+  ${p =>
+    !p.expanded &&
+    css`
+      margin-bottom: 0px;
+      border-bottom-left-radius: 0px;
+      border-bottom-right-radius: 0px;
+      border-bottom: none;
+    `}
+`;
+
+const StatusLevel = styled('div')<{level: string}>`
```

Comment:
We have these badge-like row indicators in a few places now. Should we make them a more reusable component?